app/services/anonymized_data_download_service.rb

```diff
@@ -0,0 +1,255 @@
+require 'csv'
+
+class AnonymizedDataDownloadService
+  def initialize(recipient_name, recipient_email, entity_map, entity_type)
+    @recipient_name = recipient_name
+    @recipient_email = recipient_email
+    @entity_map = entity_map
```

Comment:
This is a confusing param. What's an entity map and entity type? We should make this less opaque.

Instead of `AnonymizedDataDownloadService.new`, we could have something like this:

```ruby
AnonymizedDataDownloadService.run_for_facility(recipient_name, recipient_email, facility)
```

Reply:
@timcheadle Agreed that this approach does seem a bit hacky. I could probably do the dispatch in the Data Anonymisation Job itself and have different handlers in the service (such as `run_for_facility` and `run_for_district`?